src/main/java/Laika.java

```diff
@@ -0,0 +1,79 @@
+import java.util.Scanner;
+
+public class Laika {
+
+    public static void modifyTask(Task[] taskList, String line){
+        String[] words = line.split(" ");
+        int taskNumber = Integer.parseInt(words[1]);
+        if (words[0].equals("mark")) {
+            taskList[taskNumber - 1].markAsDone();
+            System.out.println("Laika: Good job! Task has been marked as done.");
+        } else {
+            taskList[taskNumber - 1].markAsUndone();
+            System.out.println("Laika: Alright, task has been marked as undone.");
+        }
+        System.out.println(taskList[taskNumber-1]);
+    }
+
+    public static void addTask(Task[] taskList, String line, int count){
+        if(line.startsWith("todo")){
+            taskList[count] = new Todo(line.substring(5));
+        }
+        else if (line.startsWith("deadline")) {
+            String[] words = line.split("/");
+            taskList[count] = new Deadline(words[0].substring(9),
+                    words[1].substring(3));
+        }
+        else if (line.startsWith("event")) {
+            String[] words = line.split("/");
+            taskList[count] = new Event(words[0].substring(6),
+                    words[1].substring(5),
+                    words[2].substring(3));
+        }
+        System.out.println("Laika: Gotcha! I've added the task for you\n  "
+                + taskList[count] + System.lineSeparator()
+                + "You have " + (count+1) + " tasks in your list. :P" );
+    }
+
+    public static void displayTasks(Task[] taskList,int count){
+        System.out.println("Task List:");
+        for (int i = 0; i<count;i++){
+            System.out.println((i + 1) + ") " + taskList[i]);
+        }
+    }
+
+    public static void main(String[] args) {
+        String line;
+        Scanner in = new Scanner(System.in);
```

Comment:
An alternative to using scanner could be the bufferedreader class, being more efficient at reading streams and texts.